### PR TITLE
OCPBUGS-6962: Add 'agent-installer' value to 'install_type' label

### PR DESCRIPTION
**Pull request by zaneb**
Clusters installed with an invoker value of '[agent-installer](https://github.com/openshift/installer/pull/6418)' should get an install_type of 'agent-installer'.

### CODE REVIEW

**Comment by juzhao** [comment]
LGTM, waiting for others to review

**Comment by simonpasquier** [comment]
/lgtm
/hold
waiting for approval from @PhilipGough (or someone else from RHOBS)

**Review by PhilipGough — APPROVED** [review]
/lgtm

**Pull request merged**
